docs/user-guide/docker-cli-to-kubectl.md

```diff
@@ -0,0 +1,252 @@
+<!-- BEGIN MUNGE: UNVERSIONED_WARNING -->
+
+<!-- BEGIN STRIP_FOR_RELEASE -->
+
+![WARNING](http://kubernetes.io/img/warning.png)
+![WARNING](http://kubernetes.io/img/warning.png)
+![WARNING](http://kubernetes.io/img/warning.png)
+
+<h1>PLEASE NOTE: This document applies to the HEAD of the source
+tree only. If you are using a released version of Kubernetes, you almost
+certainly want the docs that go with that version.</h1>
+
+<strong>Documentation for specific releases can be found at
+[releases.k8s.io](http://releases.k8s.io).</strong>
+
+![WARNING](http://kubernetes.io/img/warning.png)
+![WARNING](http://kubernetes.io/img/warning.png)
+![WARNING](http://kubernetes.io/img/warning.png)
+
+<!-- END STRIP_FOR_RELEASE -->
+
+<!-- END MUNGE: UNVERSIONED_WARNING -->
+# kubectl for docker users
+
+In this doc, we introduce the kubernetes command line to for interacting with the api to docker-cli users. The tool, kubectl, is designed to be familiar to docker-cli users but there are a few necessary differences. Each section of this doc highlights a docker subcommand explains the kubectl equivalent.
+
+**Table of Contents**
+<!-- BEGIN MUNGE: GENERATED_TOC -->
+- [kubectl for docker users](#kubectl-for-docker-users)
+      - [docker run](#docker-run)
+      - [docker ps](#docker-ps)
+      - [docker exec](#docker-exec)
+      - [docker logs](#docker-logs)
+      - [docker stop && docker rm](#docker-stop-&&-docker-rm)
```

Comment:
`&&` probably doesn't actually work as an anchor. Try to rebase and rerun gendocs to see if the symbol stripper removes it. Or just use `and`.